src/components/Form.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Form = () => (
+  <>
+    <form action="GET">
```

Comment:
- [ ] @josuerushanika it would be nice if you removed the action for this function, its just supposed to be a placeholder, currently it's getting an empty page as shown  below when pressed which shouldn't happen
![image](https://user-images.githubusercontent.com/61408860/224386516-80328d8b-2dfd-42a6-af86-021486eff051.png)